spec/index.bs

```diff
@@ -2595,6 +2606,9 @@ interface GPURenderPassEncoder {
     void setBlendColor(GPUColor color);
     void setStencilReference(GPUStencilValue reference);
 
+    void beginOcclusionQuery(GPUSize32 queryIndex, optional boolean preciseOcclusion = false);
```

Comment:
```suggestion
    void beginOcclusionQuery(GPUSize32 queryIndex);
```

Reply:
(not sure yet whether we want an optional argument or an optional dictionary argument)